integrations/dbolya-yolact/tutorials/sparsifying_yolact_using_recipes.md

```diff
@@ -127,11 +127,11 @@ Afterward, you will have a model that achieves roughly 28.7 mAP on the COCO data
     │   ├── dets
     │   └── scripts
     └── weights
-        └── yolact_darknet53_54_800000.pth
+        └── darknet53.pth
  ```
 
 You are ready to use the weights at `yolact/weights/yolact_darknet53_54_800000.pth` with the Neural Magic recipes to create a sparsified model.
-
+You can also download this pretrained weight directly from [here](https://drive.google.com/file/d/1dukLrTzZQEuhzitGkHaGjphlmRJOjVnP/view)
```

Comment:
can you give a bash command for this, does wget work here?

Reply:
Synced offline, Will add a sparsezoo stub Once models uploaded to make the flow easier